### PR TITLE
Proofread. Update with minor edits 2016-10-14

### PR DESCRIPTION
**Pull request by FedericoEsparza**
Sounds like the conference went well. Are the talks recorded?
 
I have a couple of additional comments/questions...

### CODE REVIEW

File: source/blog_articles/2016-10-14-online-conferences.html.markdown [hunk 1]
```diff
@@ -9,9 +9,9 @@ Gave my first ever talk at a remote online conference on Thursday.  It's been a
...
+I got a few comments and questions during the talk, which like usual hangout chat appear to have been lost to the ether.  There was a question about how time-pressed project maintainers can manage lots of pull requests, and I responded with Pat Bolger's great points on my first draft about getting the whole team involved in the review of pull requests, and of course my big favourite, "small PRs".  There wasn't extensive questioning at the end, hopefully because what I had said was transparently clear rather than just unutterably dull :-) Reassuringly I had some positive feedback by email afterwards from one audience member after.
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-14-online-conferences.html.markdown]: I'm guessing that the last line here either was intended to end with "from one audience member." or "from one audience member after another."

File: source/blog_articles/2016-10-14-online-conferences.html.markdown [hunk 2]
```diff
@@ -21,8 +21,8 @@ The [full slides from Nell's talk](http://www.slideshare.net/NellShamrell/estima
 
 ![points in days](https://www.dropbox.com/s/104oec3yghsk1im/Screenshot%202016-10-14%2010.07.14.png?dl=0)
```
Comment (FedericoEsparza) [inline source/blog_articles/2016-10-14-online-conferences.html.markdown]: This linked image isn't showing up.

**Comment by tansaku** [comment]
yes talks recorded

**Comment by tansaku** [comment]
thanks again - other comments addressed in published blog I think :-)

**Pull request merged**